substrait_consumer/conftest.py

```diff
@@ -63,7 +63,7 @@ def saveplan(request):
 
 
 PRODUCERS = [DuckDBProducer, IbisProducer, IsthmusProducer]
-CONSUMERS = [AceroConsumer, DuckDBConsumer]
```

Comment:
Does A cero not work?

Reply:
It sort of works....but most of these tests will fail because Acero doesn't support projection in the `ReadRel`

https://github.com/apache/arrow/issues/14965

I don't think there is a plan to really continue work on the Acero Consumer either.